### ISSUE TITLE
Explicit font-size:20px breaks themes

### ISSUE THREAD

**Issue by rondonjon** [wyze/atom-seti-icons#13]
I have the Metro UI theme installed (configured to 13px font size) and would like to use seti-icons with it.

Unfortunately the seti-icons package enforces font-size:20px in its stylesheet, which breaks the Metro UI's line heights and selections in the tree view:

![lineheight](https://cloud.githubusercontent.com/assets/8258823/9736879/212f9fc0-5644-11e5-9eb5-1b9be07149d4.png)

After temporarily removing the 20px in the developer panel, the seti-icons were smaller, but the other problems were solved.

Any chance to remove this setting officially and/or replacing it with "icon height = line height"?

**Comment by wyze** [comment]
I tried messing with this a bit tonight. I couldn't come up with a solution.

Once the icons were smaller, they were then misaligned. I think this would be too large of a change that I am not sure how I would approach it. Even if I added a setting to allow you to force all icons to be the same size, it would have the same affect.

In the `ui-mixins.less` file, each icon can send in their own `font-size`, and if you look at `icons.less`, that is exactly what is happening.

<img width="118" alt="screen shot 2015-09-10 at 9 18 43 pm" src="https://cloud.githubusercontent.com/assets/186971/9805674/9574e89a-5801-11e5-90b9-8b345bdc1d8a.png">

If you would like to take a stab at it, you're more than welcome to send a PR over.
